radix-engine/src/blueprints/resource/resource_manager_common.rs

```diff
@@ -146,12 +146,15 @@ fn build_access_rules(
         DenyAll,
     );
     vault_access_rules.set_group_and_mutability(
-        MethodKey::new(NodeModuleId::SELF, VAULT_TAKE_NON_FUNGIBLES_IDENT),
+        MethodKey::new(
```

Comment:
It feels like each vault should have its own, blueprint (fungible/non-fungible) dependent?

But anyway, under the proposal, this mapping would live in the blueprint/package anyway.

Reply:
Yup, don't think this is too big a deal.